app/views/sign_up/registrations/_required_pii_accordion.html.slim

```diff
@@ -0,0 +1,31 @@
+.mt4
+  = accordion('you-will-need', t('devise.registrations.start.accordion')) do
+    .clearfix
+      .col.col-1.center
+        = image_tag(asset_url('get-started/email-password.svg'),
+          size: '40', alt: '', class: 'mt1')
+      .col.col-11.pl2
+        p.mb3 = t('devise.registrations.start.bullet_1_html')
+    .clearfix
+      .col.col-1.center
+        = image_tag(asset_url('get-started/2FA.svg'), size: '40', alt: '', class: 'mt1')
+      .col.col-11.pl2
+        p.mb3 = t('devise.registrations.start.bullet_2_html')
+    .clearfix
+      .col.col-1.center
+        = image_tag(asset_url('get-started/personal-details.svg'),
+          size: '40', alt: '', class: 'mt1')
```

Comment:
@zachmargolis Actually, I believe these fall under the category of decorative images, which should have `alt=''`. Here is a link to the spec https://www.w3.org/WAI/tutorials/images/decorative/